Sources/App/Resources/en.lproj/Localizable.strings

```diff
@@ -172,7 +172,7 @@
 "improv.wifi.alert.description" = "Please enter your SSID and password.";
 "improv.wifi.alert.password_placeholder" = "Password";
 "improv.wifi.alert.ssid_placeholder" = "Network Name";
-"improv.wifi.alert.title" = "Connect to WiFi";
```

Comment:
This string key was already sent to lokalise for translation, please create a new one so it is translated as well.